WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/FollowerTotalsUseCase.kt

```diff
@@ -86,21 +85,21 @@ class FollowerTotalsUseCase
         forced: Boolean,
         fetchMode: PagedMode
     ): State<Map<FollowerType, Int>> {
-        val deferredWpComResponse = GlobalScope.async(bgDispatcher) {
```

Comment:
I have no idea why this was running in the GlobalScope but it definitely doesn't have to.